libs/fn/src/lib/segmented-button/segmented-button.component.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
I think this can be moved directly to component decorator since it's just a one-liner